python/tank/authentication/interactive_authentication.py

```diff
@@ -73,7 +73,7 @@ def _get_qt_state():
         from .ui.qt_abstraction import QtGui, QtCore
         qt_core = QtCore
         qt_gui = QtGui
-        qapp_instance_active = (QtGui.QApplication.instance() is not None)
+        qapp_instance_active = (QtGui.qApp is not None)
```

Comment:
I was trying out something, turns out it wasn't needed. I'll remove that change actually just in case (paranoia!)